_apidocs/searchgov-results/v2/openapi.yml

```diff
@@ -63,6 +63,13 @@ paths:
             type: string
             format: string
             default: relevance
+        - name: sitelimit
+          in: query
+          description: 'Allows you to limit the results by subdomains or subfolders. You can limit to multiple locations using a space-separated list (ex. `sitelimit=search.gov/about search.gov/get-started`)'
```

Comment:
Super important item I just remembered - the value passed in the sitelimit must be in scope of the Domains list in the admin center.